templates/ssm_sensitive_values_seqerakit.json

```diff
@@ -35,7 +35,7 @@
 		"ssm_key": 	"/seqera/tower-dev/seqerakit/codecommit-password",
 		"value": 	"CHANGE_ME_IF_NECESSARY"
 	},
-    "TOWER_CODECOMMIT_REGION": { 
+    "TOWER_CODECOMMIT_BASEURL": { 
 		"ssm_key": 	"/seqera/tower-dev/seqerakit/codecommit-region",
```

Comment:
Initial reaction was that these `ssm_key` paths could be changed as well for alignment on nomenclature but looking for a sanity check that this would be a breaking change and not to be messed with.